Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+    
+    10
+    15
+
+### [4. slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)
+
+**Description:**
+Returns the selected elements in an array, as a new array object and selects the elements starting at the given start argument, and ends at the given end argument(excluding end argument).
+
+**Syntax:**
+ * _array.slice(start, end)_
+
+ | Parameter | Description |
+ | :-------- | :---------- |
+ | start     | Optional. specifies starting index of the selection    |
+ | end       | Optional. specifies ending index of the selection    |
+
+ **Return value:**
+ New array containing the selected elements.
+
+ **Example:**
+```
+    let arr = [23,56,87,32,75,13];
+    let new_arr = arr.slice();
+    console.log(arr);
+    console.log(new_arr);
+```    
+*Output:*
+
+    [23,56,87,32,75,13]
+    [23,56,87,32,75,13]
+
+### [5. splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)
+
+**Description:**
+Adds/removes items to/from an array, and returns the removed item(s).
+
+**Syntax:**
+ * _array.splice(index, howmany, item1, ....., itemX)_
+
+| Parameter | Description|
+| :-------- | :--------- |
+| index	    | Required. An integer that specifies at what position to add/remove items |
+| howmany | Optional. The number of items to be removed.|
+| item1, ..., itemX | Optional. The new item(s) to be added to the array |
+
+**Return value:**
+Array without the removed items.
+
+**Example:**
+```
+const languages = ['C++', 'Java', 'Html', 'Python', 'C']; 
+console.log(languages); 
+// Add 'Julia' and 'Php' after removing 'Html'. 
+const removed = languages.splice(2, 1, 'Julia', 'Php') 
+console.log(languages); 
+console.log(removed); 
+```
+*Output:*
+
+    C++,Java,Html,Python,C
+    C++,Java,Julia,Php,Python,C
+
+### [6. concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)
+
+**Description:**
+used to join two or more arrays.
+
+**Syntax:**
+ *  _array1.concat(array2, array3, ..., arrayX)_
+
+| Parameter | Description |
+| :-------- | :---------- |
+| array2, array3, ..., arrayX | Required. The arrays to be joined. |
+
+**Return value:**
+joined array
+
+**Example:**
+```
+const num1 = [11, 12, 13],
+num2 = [14, 15, 16],
+num3 = [17, 18, 19];
+console.log(num1.concat(num2, num3));
+```
+*Output:*
+
+    [11,12,13,14,15,16,17,18,19]
+
+### [7. join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)
+
+**Description:**
+Returns the array as a string. The elementsare separated by a specified separator. The default separator is comma (,).
+
+**Syntax:**
+ * _array.join(separator)_
+
+| Parameter | Description |
+| :-------- | :---------- |
+| separator	| Optional.   | 
+
+**Return value:**
+String, representing the array values, separated by the specified separator.
+
+**Example:**
+```
+const elements = [\'geeksforgeeks\', \'gfg\']; console.log(elements.join()); 

```

Comment:
Please remove the escaping on the strings. Also please use the same data from above, the list of languages.
Again, do not use console.log and just list the output

Reply:
Got it.